### DISCUSSION TITLE
How to share code changes with fellow developer

### DISCUSSION THREAD

**Discussion by hiteshnsharma** [grandnode/grandnode2#339] [Q&A]
Hi Team,

I want to use GrandNode2 for my project. I will be making some customization to the product (like adding/removing product properties). How can I share these changes with a fellow developer who will also be working on this project. 
Since these properties are stored in database, so these will not be tracked by version control.

**Comment by zaLTys** [comment]
Share migration scripts / adjust installation / share same mongo instance for dev?
